templates/hoover-deps.nomad

```diff
@@ -38,6 +38,7 @@
         xpack.monitoring.collection.index.stats.timeout = "30s"
         xpack.monitoring.collection.index.recovery.timeout = "30s"
         xpack.monitoring.history.duration = "32d"
+        discovery.seed_hosts = "{% raw %}${attr.unique.network.ip-address}{% endraw %}"
```

Comment:
Please don't set this here, since it's also overridden in the job descriptions for `es-data`. If you need this config on the master node, set it explicitly there, not in this macro.